projects/valtimo/components/src/lib/components/page-title/page-title.component.ts

```diff
@@ -141,4 +142,13 @@ export class PageTitleComponent implements OnInit, OnDestroy, AfterViewInit {
       })
     );
   }
+
+  private openCompactModeSubscription(): void {
+    this._subscriptions.add(
+      this.pageHeaderService.compactMode$.subscribe(compactMode => {
+        this.isCompact = compactMode;
+        this.compactMode = compactMode;
```

Comment:
Why do you have both isCompact and compactMode if they get assigned the same value?